sub_workflows/kfdrc_annot_vcf_sub_wf.cwl

```diff
@@ -36,15 +36,25 @@ outputs:
   annotated_public_maf: {type: File, outputSource: kfdrc_vcf2maf_public/output_maf}
 
 steps:
+  bcftools_strip_info:
+    run: ../tools/bcftools_strip_ann.cwl
+    in:
+      input_vcf: input_vcf
+      output_basename: output_basename
+      tool_name: tool_name
+      strip_info: bcftools_annot_columns
+    out: [stripped_vcf]
+
```

Comment:
My understanding of this is that whatever fields we are planning to add (`bcftools_annot_columns`) we first strip out of the single-caller VCFs if present. So that includes INFO/AF but is potentially more general. Is that right?

Reply:
Correct! In the future if we want overlapping annotations, we can change this, but as we have inadvertently learned, that can have nasty consequences. The desired behavior I believe is that if you annotate an existing field, you should replace it.